data/experiment-recipe-samples/desktop-113-localizations.json

```diff
@@ -0,0 +1,87 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "branches": [
+    {
+      "feature": {
+        "featureId": "unused-feature-id-for-legacy-support",
+        "enabled": false,
+        "value": {}
+      },
+      "features": [
+        {
+          "featureId": "nimbus-qa-1",
+          "value": {
+            "value": {
+              "$l10n": {
+                "id": "control-text",
+                "text": "English text",
+                "comment": "Text for the control branch."
+              }
```

Comment:
Wait what is this structure?  I thought it was just the string value is the identifier?

Reply:
No this came up during proposal review. We need all this information to generate the en_US fluent files for pontoon.